charts/trino/templates/service.yaml

```diff
@@ -14,6 +14,9 @@ spec:
       targetPort: http
       protocol: TCP
       name: http
+      {{- if .Values.service.nodePort }}{{- if eq .Values.service.type "NodePort" }}
```

Comment:
I don't think the condition on `.Values.service.type` is required. It's better to have the deployment fail with an actionable error from Kubernetes (that you can't set a `nodePort` for a non-node-port service type), than silently ignore a value that has been set explicitly. It makes it harder to debug, and we don't know what which service type should be used.

Reply:
Hi, I removed the "NodePort" type check. I also tested with the ClusterIP type and assigning a nodePort, and kubernetes returns the error:
`* Service "my-release-trino" is invalid: spec.ports[0].nodePort: Forbidden: may not be used when `type` is 'ClusterIP'`